tests/integration/dbapi/async/V2/test_queries_async.py

```diff
@@ -408,7 +427,7 @@ async def test_bytea_roundtrip(
         ), "Invalid bytea data returned after roundtrip"
 
 
-@fixture(scope="session")
```

Comment:
This fixture failed for me since it's using `connection_system_engine_no_db`, which itself is per test scoped. This causes pytest to fail. I'm not sure how it used to work and if it's related to my changes. I'll take a look

Reply:
This is the error I get locally on the main now: `ScopeMismatch: You tried to access the function scoped fixture connection_system_engine_no_db with a session scoped request object, involved factories:`

Reply:
It works on github CI because currently the use_database test (which is the one that uses this fixture) is skipped for staging

Reply:
I see. That's annoying :(
My worry here is if we reuse this fixture it might run into race condition when one test is trying to delete a db and another one to use it. Maybe let's add a few random numbers just to be safe?